doc/tor.md

```diff
@@ -92,13 +93,9 @@ for normal IPv4/IPv6 communication, use:
 To test your set-up, you might want to try connecting via Tor to just a
 a single Zcash hidden server.
 
-Edit zcash.conf and comment out any nodes you might normally connect to.
+Launch zcashd as follows, connecting to the Zcash hidden server:
 
-	#addnode=betatestnet.z.cash
-
-Launch zcashd as follows, adding the Zcash hidden server:
-
-	zcashd -onion=127.0.0.1:9050 -addnode=zctestseie6wxgio.onion
+	./zcashd -onion=127.0.0.1:9050 -connect=zctestseie6wxgio.onion
```

Comment:
There are two scenarios we should document:
1. You want to connect to just a single node, in which cause use -connect.
2. You want to connect to multiple tor nodes, -addnode=.. -dnsseed=0 -onlynet=onion (might be -onlynet=tor)